### ISSUE TITLE
// inside of a template literal is incorrectly parsed as a comment

### ISSUE THREAD

**Issue by vjeux** [prettier/prettier#1539]
```js
`${a}`;
`http://dev.apollodata.com/`;
```

```js
{ type: 'Line',
  value: 'dev.apollodata.com/`;',
```

```js
test.js: Error: Comment location overlaps with node location
    at decorateComment (/Users/vjeux/random/prettier/src/comments.js:117:11)
    at decorateComment (/Users/vjeux/random/prettier/src/comments.js:91:7)
    at comments.forEach (/Users/vjeux/random/prettier/src/comments.js:137:5)
    at Array.forEach (native)
    at Object.attach (/Users/vjeux/random/prettier/src/comments.js:136:12)
    at attachComments (/Users/vjeux/random/prettier/index.js:51:14)
    at format (/Users/vjeux/random/prettier/index.js:81:23)
    at formatWithShebang (/Users/vjeux/random/prettier/index.js:91:12)
    at Object.format (/Users/vjeux/random/prettier/index.js:105:12)
    at format (/Users/vjeux/random/prettier/bin/prettier.js:234:19)
```

**Comment by JamesHenry** [comment]
Related issue: https://github.com/eslint/typescript-eslint-parser/issues/216

**Comment by bakkot** [comment]
This was fixed by #1578, I believe.
